demos/python_demos/face_recognition_demo/face_recognition_demo.py

```diff
@@ -55,6 +56,8 @@ def build_argparser():
                          help="(optional) Crop the input stream to this height " \
                          "(default: no crop). Both -cw and -ch parameters " \
                          "should be specified to use crop.")
+    general.add_argument('-match_algo', default='HUNGARIAN', choices=MATCH_ALGO,
```

Comment:
Long options should start with two dashes.

Reply:
i add '--match_algo' for long option, thanks.